pxtcompiler/emitter/decompiler.ts

```diff
@@ -541,6 +546,10 @@ ${output}</xml>`;
             write(`</block>`)
         }
 
+        function emitWorkspaceComment(text: string) {
+            write(`<comment h="120" w="160">${text}</comment>`);
```

Comment:
You need to HTML escape the text. Otherwise if an invalid character is placed in the comment then Blockly will fail to load the workspace from the XML

Reply:
Done.